ouroboros-network-api/src/Ouroboros/Network/SizeInBytes.hs

```diff
@@ -0,0 +1,5 @@
+module Ouroboros.Network.SizeInBytes (SizeInBytes) where
```

Comment:
Seems like we could upstream this type definition to `cardano-base`, e?

Reply:
Yes, it makes sense.  Should it be a newtype? Then we could control which instances it has.

Reply:
I don't have a strong preference, but I would lean `newtype` myself.

Reply:
I'm going to Approve without this being resolved.

Reply:
This deserves a separate PR, it will be every so slightly easier when rebasing other PRs over these changes.